addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -335,6 +336,30 @@ Wysiwyg.include({
         this.odooEditor.observerActive("toggleMegaMenu");
         return this.snippetsMenu.activateSnippet($(megaMenuEl));
     },
+    /**
+     * Restores all the carousels so their first slide is the active one.
+     *
+     * @private
+     */
+    _restoreCarousels() {
+        const carouselEls = this.$editable[0].querySelectorAll(".carousel");
+        carouselEls.forEach(carouselEl => {
+            // Set the first slide as the active one.
+            carouselEl.querySelectorAll(".carousel-item").forEach((itemEl, i) => {
+                itemEl.classList.remove("next", "prev", "left", "right", "active");
+                if (i === 0) {
+                    itemEl.classList.add("active");
+                }
+            });
+            carouselEl.querySelectorAll(".carousel-indicators li[data-slide-to]").forEach((indicatorEl, i) => {
+                indicatorEl.classList.remove("active");
+                indicatorEl.replaceChildren();
```

Comment:
What's the point of this one? Probably needs a comment if this is useful.

Reply:
I added it to replace the `.empty()` call that was in the  `cleanForSave` :slightly_smiling_face: I just translated the jQuery to JS (in order to not add more jQuery), but I am not sure why it was needed: I kept it just in case :+1:

Reply:
Ok :+1: Then maybe add a comment to say it is probably useless and can be removed in master? :pray:

Reply:
In the end I removed it, as it will contradict with the "play" icon that will be added in the "video in image gallery" PR (unless you don't agree with it in that other PR, but it depends on which one will be merged first)